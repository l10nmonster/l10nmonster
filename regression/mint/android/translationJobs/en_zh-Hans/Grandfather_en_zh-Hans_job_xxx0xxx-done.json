{
	"sourceLang": "en",
	"targetLang": "zh-Hans",
	"translationProvider": "Grandfather",
	"jobGuid": "xxx0xxx",
	"status": "done",
	"tus": [
		{
			"guid": "Mi8CGeTkNxfnO-OshrNPE4_FrkVbQajzADA09g5O2Zk",
			"ntgt": [
				"冬天就",
				{
					"t": "bx",
					"v": "<b>"
				},
				"要来了",
				{
					"t": "ex",
					"v": "</b>"
				}
			],
			"ts": 1,
			"q": 70
		}
	],
	"updatedAt": "2022-05-29T00:00:00.000Z"
}