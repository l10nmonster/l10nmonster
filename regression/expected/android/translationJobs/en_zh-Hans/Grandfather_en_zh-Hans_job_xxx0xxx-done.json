{
	"sourceLang": "en",
	"targetLang": "zh-Hans",
	"translationProvider": "Grandfather",
	"jobGuid": "xxx0xxx",
	"status": "done",
	"tus": [
		{
			"guid": "Mi8CGeTkNxfnO-OshrNPE4_FrkVbQajzADA09g5O2Zk",
			"ntgt": [
				"冬天就",
				{
					"t": "bx",
					"v": "<b>"
				},
				"要来了",
				{
					"t": "ex",
					"v": "</b>"
				}
			],
			"ts": 1,
			"q": 70
		},
		{
			"guid": "Kexjq-2WakTgEVK2nZbiSlWPNJ_dQ1_Cll0vWPnNzVg",
			"ntgt": [
				"房子: ",
				{
					"t": "x",
					"v": "%1$s"
				},
				" 座右铭: ",
				{
					"t": "x",
					"v": "%2$s"
				}
			],
			"ts": 1,
			"q": 70
		},
		{
			"guid": "DZAA56iaQf0fJvMG8fv3FgBCiGkUGweBDBEJunOp5zw",
			"ntgt": [
				"I know 100%%"
			],
			"ts": 1,
			"q": 70
		},
		{
			"guid": "Kexjq-2WakTgEVK2nZbiSlWPNJ_dQ1_Cll0vWPnNzVg",
			"ntgt": [
				"房子: ",
				{
					"t": "x",
					"v": "%1$s"
				},
				" 座右铭: ",
				{
					"t": "x",
					"v": "%2$s"
				}
			],
			"ts": 1,
			"q": 70
		},
		{
			"guid": "oRdDrqs0lf4rPJdRA5ut9PI_ix2KZmBaqiDiyAnT9XU",
			"ntgt": [
				"Translation-Frozen: ",
				{
					"t": "bx",
					"v": "<strike>"
				},
				{
					"t": "x",
					"v": "%1$s"
				},
				{
					"t": "ex",
					"v": "</strike>"
				}
			],
			"ts": 1,
			"q": 70
		},
		{
			"guid": "GxywI1j2v6A2Bq8oomjIkF6I2KphYzxSgKDNxWCHbJM",
			"ntgt": [
				"Translation-The translation has a nbsp, source doesn't"
			],
			"ts": 1,
			"q": 70
		}
	],
	"updatedAt": "2022-05-29T00:00:00.000Z"
}